### DISCUSSION TITLE
Textures do not load on custom entities

### DISCUSSION THREAD

**Discussion by bschurman1** [bridge-core/bridge.#16] [Q&A] [answered]
Title. I’m making an addon, so i obviously need textures. The behavior pack side of things for my entity works, but my resources don’t, even though I have everything set up correctly (I’ve compared to other resource packs)

**Comment by solvedDev** [comment, marked as answer]
Could you send a screenshot of the file you've created? It's definitely not an issue with "bridge." itself , rather an issue with how you've created the entity file
